sep-service/ViewModels/Address.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace sep_service.ViewModels
```

Comment:
Are these models auto-generated? 
My only comment here would be to try to have the namespace conform better to C# naming guidelines: 
* `sep_service.ViewModels` --> `SepService.ViewModels`

Thoughts?

Reply:
It's auto generated, but I can adjust it.